packages/expo-updates/ios/EXUpdates/Database/EXUpdatesDatabase.m

```diff
@@ -14,7 +14,7 @@ @interface EXUpdatesDatabase ()
 @end
 
 static NSString * const EXUpdatesDatabaseErrorDomain = @"EXUpdatesDatabase";
-static NSString * const EXUpdatesDatabaseFilename = @"expo-v3.db";
+static NSString * const EXUpdatesDatabaseFilename = @"expo-v4.db";
```

Comment:
With the new database file, expo-updates is going to try to reinstall updates and potentially overwrite files -- is this handled gracefully?

Also, we're soon likely going to need a migration system that makes incremental schema changes since resetting the database on every major version bump will cause the app not to honor older manifest filters and server-defined headers that were in the old database. And with migrations, we wouldn't have old .db files lying around until the app is uninstalled.